modules/simple-payments/simple-payments.php

```diff
@@ -86,23 +88,26 @@ function parse_shortcode( $attrs, $content = false ) {
 
 	function output_shortcode( $data ) {
 		$items = '';
+		$cssPrefix = self::$cssClassnamePrefix;
+
 		if ( $data['multiple'] ) {
-		       $items="<div class='jetpack-simple-payments-items'>
-		       <input class='jetpack-simple-payments-items-number' type='number' value='1' id='{$data['dom_id']}_number'>
-		       </div>";
+			$items="<div class='${cssPrefix}-items'>
+				<input class='${cssPrefix}-items-number' type='number' value='1' id='{$data['dom_id']}_number' />
+			</div>";
 		}
-		$output = "
-<div class='{$data[ 'class' ]} jetpack-simple-payments__wrapper'>
-	<p class='jetpack-simple-payments__purchase-message'>
-	</p>
-	<div class='jetpack-simple-payments__title'>{$data['title']}</div>
-	<div class='jetpack-simple-payments__description'>{$data['description']}</div>
-	<div class='jetpack-simple-payments__price'>{$data['price']}</div>
-	{$items}
-	<div class='jetpack-simple-payments__button' id='{$data['dom_id']}_button'></div>
-</div>
-";
-		return $output;
+
+		return "
```

Comment:
This will make tabs in site html code as well, it was tabbed to the left so we dont print out unnecessary tabs